src/Pos/Wallet/Web/Tracking.hs

```diff
@@ -273,7 +273,6 @@ applyModifierToWSet
     -> m ()
 applyModifierToWSet wAddr newTip mapModifier = do
     -- TODO maybe do it as one acid-state transaction.
-    mapM_ WS.removeWAddress (MM.deletions mapModifier)
```

Comment:
Simply ignoring deletions seem like dirty hack -- summoning @pva701 for an opinion about how to do it better.